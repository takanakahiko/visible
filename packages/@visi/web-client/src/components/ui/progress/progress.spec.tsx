import { render } from '@testing-library/react';
import React from 'react';

import { Progress } from '.';

describe('Progress', () => {
  it('matches snapshot', () => {
    const { container } = render(<Progress max={10} value={5} label="foo" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
