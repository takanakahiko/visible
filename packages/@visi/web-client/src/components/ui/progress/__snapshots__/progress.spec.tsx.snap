// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress matches snapshot 1`] = `
<div
  class="inline-flex w-full items-center"
>
  <div
    class="rounded-full h-2 bg-gray-300 flex-grow mr-2"
  >
    <div
      aria-hidden="true"
      class="bg-primary-500 rounded-full h-full"
      style="width: 50%;"
    />
    <progress
      class="sr-only"
      max="10"
      value="5"
    />
  </div>
  <span
    aria-hidden="true"
    class="text-sm text-gray-600"
  >
    50
    %
  </span>
</div>
`;
